consul-connect/index.json

```
@@ -40,10 +46,10 @@
   "environment": {
     "uilayout": "terminal",
     "uimessage1":
-      "\u001b[32mYour Interactive Bash Terminal. A safe place to learn and execute commands version 0.1.\u001b[m\r\n"
+      "\u001b[32mYour Interactive Bash Terminal. A safe place to learn and execute commands.\u001b[m\r\n"
   },
   "backend": {
-    "imageid": "hashicorp-hashistack",
+    "imageid": "ubuntu",
```

```suggestion
    "imageid": "hashicorp-hashistack",
```
If you use the pre-baked image, you shouldn't have to install Consul separately in the setup script, which will speed up boot time.

Maybe we should also put the dashboard and counting services into the basic image, or make a separate Consul image as well? I'll ask @acidprime for an opinion.

---

Yes, the main issue I had with that is we cannot test it "locally". So I based it on Ubuntu and added the extra steps necessary to make it work on a free profile.

### Education approach
We could make provision more modular for now (like having a separate script to make ubuntu image adhere to the environment) and comment out that part of the provision once we merge on this repo.

### Practitioner Approach
As a practitioner having the possibility to see behind the scenes how "easy" is to setup our tools seems something we want to promote rather than hide behind a prepared, even if faster, environment.

---

@danielehc FYI I do have a WIP docker workflow for testing this locally 

https://github.com/hashicorp/katakoda/tree/master/environments/hashistack